src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMStep.java

```diff
@@ -0,0 +1,248 @@
+                                                 @QueryParameter String projectName) {
+            return formValidation.doCheckProjectName(serverId, credentialsId, projectName);
+        }
+
+        @Override
+        @POST
+        public FormValidation doCheckRepositoryName(@QueryParameter String serverId,
+                                                    @QueryParameter String credentialsId,
+                                                    @QueryParameter String projectName,
+                                                    @QueryParameter String repositoryName) {
+            return formValidation.doCheckRepositoryName(serverId, credentialsId, projectName, repositoryName);
+        }
+
+        @Override
+        @POST
+        public FormValidation doCheckServerId(@QueryParameter String serverId) {
+            return formValidation.doCheckServerId(serverId);
+        }
+
+        @Override
+        @POST
+        public ListBoxModel doFillCredentialsIdItems(@QueryParameter String baseUrl,
+                                                     @QueryParameter String credentialsId) {
+            return formFill.doFillCredentialsIdItems(baseUrl, credentialsId);
+        }
+
+        @Override
+        @POST
+        public HttpResponse doFillProjectNameItems(@QueryParameter String serverId,
+                                                   @QueryParameter String credentialsId,
+                                                   @QueryParameter String projectName) {
+            return formFill.doFillProjectNameItems(serverId, credentialsId, projectName);
+        }
+
+        @Override
+        @POST
+        public HttpResponse doFillRepositoryNameItems(@QueryParameter String serverId,
+                                                      @QueryParameter String credentialsId,
+                                                      @QueryParameter String projectName,
+                                                      @QueryParameter String repositoryName) {
+            return formFill.doFillRepositoryNameItems(serverId, credentialsId, projectName, repositoryName);
+        }
+
+        @Override
+        @POST
+        public ListBoxModel doFillServerIdItems(@QueryParameter String serverId) {
+            return formFill.doFillServerIdItems(serverId);
+        }
+
+        public Optional<BitbucketScmHelper> getBitbucketScmHelper(@Nullable String serverId, @Nullable String credentialsId) {
+            return bitbucketPluginConfiguration.getServerById(serverId)
+                    .map(serverConf -> new BitbucketScmHelper(bitbucketClientFactoryProvider, serverConf, credentialsId));
+        }
+
+        @Override
+        public List<GitSCMExtensionDescriptor> getExtensionDescriptors() {
+            return emptyList();
+        }
+
+        @Override
+        public String getFunctionName() {
+            return "bbs";
```

Comment:
Shouldn't this be a verb? Like in all other implementation of this function. Example "archive", "unarchive"

Reply:
There are plenty that aren't too - like `pwd` and `tool`